grassroots-backend/src/mikro-orm.config.ts

```diff
@@ -2,23 +2,46 @@
 import { defineConfig, PostgreSqlDriver } from "@mikro-orm/postgresql";
 import { UserEntity } from "./users/User.entity";
 import { ContactEntity } from "./contacts/entities/Contact.entity";
-
-import dotenvFlow from "dotenv-flow";
-import { getEnvFilePaths } from "./GetEnvFilePaths";
 import { OrganizationEntity } from "./organizations/Organization.entity";
-dotenvFlow.config({
-  // Reversed so that earlier files take priority, to align with the ConfigModule.
-  files: getEnvFilePaths().reverse(),
-});
+import * as dotenv from "dotenv";
+import { readFileSync, existsSync } from "fs";
+import { getEnvFilePaths } from "./GetEnvFilePaths";
+
+/**
+ * Loads and combines environment variables from multiple .env files
+ * Earlier files in the list take priority (same as dotenv-flow behavior)
+ */
+function loadEnvironmentVariables(): Record<string, string> {
+  // Reversed so that earlier files take priority, to align with the ConfigModule
+  const envFiles = getEnvFilePaths().reverse();
+  let combinedEnv: Record<string, string> = {};
+
+  for (const filePath of envFiles) {
+    if (existsSync(filePath)) {
+      try {
+        const fileContent = readFileSync(filePath, "utf8");
+        const parsedEnv = dotenv.parse(fileContent);
+        combinedEnv = { ...combinedEnv, ...parsedEnv };
+      } catch (error) {
+        console.warn(`Warning: Could not read env file ${filePath}:`, error);
```

Comment:
I think we should probably throw in this case. If there an env file there but it can't be parsed, that implies it's actually broken somehow.